### PR TITLE
Add new handlers to get workbooks and collections list by ids

### PR DESCRIPTION
**Pull request by Sergey-weber**

### CODE REVIEW

File: src/controllers/collections.ts [hunk 1]
```diff
@@ -65,6 +66,21 @@ export default {
         res.status(code).send(response);
     },
 
+    getCollectionsListByIds: async (req: Request, res: Response) => {
+        const {body} = req;
+
+        const result = await getCollectionsListByIds(
+            {ctx: req.ctx},
+            {
+                collectionIds: body.collectionIds,
```
Comment (jhoncool) [inline src/controllers/collections.ts:75]: I think we need to set some limit for `collectionIds` and `workbookIds` sizes. For example 1000

File: src/services/new/collection/get-collections-list-by-ids.ts [hunk 2]
```diff
@@ -56,30 +57,70 @@ export const getCollectionsListByIds = async (
...
+                permission: isEnabledFeature(ctx, Feature.UseLimitedView)
+                    ? CollectionPermission.LimitedView
+                    : CollectionPermission.View,
+            })
+            .then(() => {
+                acceptedCollectionsMap.set(collection.model, parentIds);
+
+                return collection;
+            })
+            .catch(() => {});
```
Comment (jhoncool) [inline src/services/new/collection/get-collections-list-by-ids.ts:94]: So we can return not all list? Is it okay?

Comment (Sergey-weber) [reply src/services/new/collection/get-collections-list-by-ids.ts:96]: yes, also added tests for these cases, it by design

File: src/controllers/collections.ts (hunk 1 above)
Comment (Sergey-weber) [reply src/controllers/collections.ts:75]: ok, added limits to validation

**Pull request merged**